mobile/src/people/departments/departments-horizontal-scrollable-list.tsx

```diff
@@ -30,12 +32,80 @@ export class DepartmentsHScrollableList extends Component<DepartmentsHScrollable
         }
     }
 
+    public shouldComponentUpdate(nextProps: DepartmentsHScrollableListProps) {
+        if (this.currentPage > this.employeeCards.length) {
```

Comment:
it looks like a copy-paste in these two branches. Could you please refactor that?

Reply:
addressed